lib/parse/index.js

```diff
@@ -0,0 +1,22 @@
+/*!
+ * parse/index.js - JOSE Parser Entry Point
+ *
+ * Copyright (c) 2015 Cisco Systems, Inc. See LICENSE file.
+ */
+"use strict";
+
+var compact = require("./compact"),
+    json = require("./json");
+
+var parse = module.exports = function(input) {
+  if ("string" === typeof input) {
```

Comment:
I thought about that, but this is meant to be the JOSE structures (JWE or JWS).  I'm not sure how likely it is for someone to get those as Buffers, and the conversion is fairly trivial (input.toString()).

But since it is fairly trivial, I suppose we can:
if (Buffer.isBuffer(input)) { input = input.toString(); }